src/modules/global/components/RoutePrivateGuard.tsx

```diff
@@ -1,19 +1,14 @@
 import React, { PropsWithChildren } from "react";
-import { Navigate, useLocation } from "react-router-dom";
+import { Navigate } from "react-router-dom";
 import { useRecoilValue } from "recoil";
-import { pathSelector } from "../state/navigationState";
+import { loggedState } from "../../authentification/state";
 
 export const RoutePrivateGuard: React.FC<PropsWithChildren> = ({
   children,
 }) => {
-  const filteredRoutesArray = useRecoilValue(pathSelector);
-  const pathsArray = filteredRoutesArray.map((route) => {
-    return route.path;
-  });
+  const isLoggedIn = useRecoilValue(loggedState);
 
-  const { pathname } = useLocation();
-
-  if (!pathsArray.includes(pathname)) return <Navigate to="/" />;
+  if (!isLoggedIn) return <Navigate to="/" />;
```

Comment:
You should navigate him to login or?

Reply:
Changed path to navigate to a login page.